sensor_msgs/Image color_img
sensor_msgs/Image depth_img

---
sensor_msgs/PointCloud2[] pcds

